### PR TITLE
Update coq-fcsl-pcm and release v1.2.0

### PR DESCRIPTION
**Pull request by anton-trunov**

### CODE REVIEW

File: released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.0.0/opam [hunk 1]
```diff
@@ -1,5 +1,4 @@
 opam-version: "2.0"
-name: "coq-fcsl-pcm"
 version: "1.0.0"
```
Comment (clarus) [inline released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.0.0/opam]:
```suggestion
```
(like for name)

File: released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.1.0/opam [hunk 2]
```diff
@@ -1,5 +1,4 @@
 opam-version: "2.0"
-name: "coq-fcsl-pcm"
 version: "1.1.0"
```
Comment (clarus) [inline released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.1.0/opam]:
```suggestion
```

File: released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.2.0/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+version: "1.2.0"
```
Comment (clarus) [inline released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.2.0/opam]:
```suggestion
```

File: released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.2.0/opam [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+depends: [
+  "ocaml"
+  "coq" {(>= "8.10" & < "8.12~") | (= "dev")}
+  "coq-mathcomp-ssreflect" {(>= "1.10.0" & < "1.11~") | (= "dev")}
+]
+tags: [
+  "keyword:separation logic"
+  "keyword:partial commutative monoid"
+  "category:Computer Science/Data Types and Data Structures"
+  "logpath:fcsl"
```
Comment (clarus) [inline released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.2.0/opam]:
```suggestion
  "logpath:fcsl"
  "date:2019-11-07"
```

**Comment by anton-trunov** [comment]
@clarus Thank you for all the suggestions. I have a question though, do we still keep `version: "dev"` despite the fact the file name has this information?

**Comment by clarus** [comment]
@anton-trunov Np! Yes you are right, ideally it should be changed also. For me that is less critical as this is a development package.

**Comment by anton-trunov** [comment]
I see, thanks. Next time I'll try to remember remove that too :)

**Pull request merged**